### DISCUSSION TITLE
Scroll to element once the element has been rendered

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#13942] [General - Components / Directives / etc]
Hello,

I have the following issue with the scrolling: 

The following card is in a for loop:
```
<q-card @hook:mounted="dayIsReady((weekIndex - 1) * 7 + index + 1)" > ... </q-card>
```
I am calling the method dayIsReady which contains the scroll:

const ele = document.getElementById("scroll");
let target = getScrollTarget(ele);
setScrollPosition(target, 500, 1000);

I checked that once the element is loaded, scroller is not working. 

If I put it somewhere else, e.g. to mounted() { ...} it works fine.

I also tried nextTick but it did not solve the issue. Is there some way to wait for the scroller to make sure the element exists before it triggers?

**Comment by pdanpdan** [comment]
Please make a codepen, because of you are using Qv1/Vue 2 it should be triggered when it's mounted. Maybe your `#scroll` element is not available?

**Comment by i-mage** [comment]
It is triggered when mounted. Only when I try to trigger it once the element which has been loaded dynamically has finished loading it doesn't work. The dayIsReady method is fired though.
